app/nexus.go

```diff
@@ -0,0 +1,545 @@
+		}
+	}
+
+	if len(updatedPolicySpecs) == len(existingPolicySpecs) {
+		return nil, fmt.Errorf("no updates to be made")
+	}
+	existingEndpoint.Spec.PolicySpecs = updatedPolicySpecs
+	return c.callUpdateEndpoint(existingEndpoint, resourceVersion, asyncOperationId)
+}
+
+func (c *NexusClient) deleteEndpoint(
+	endpointID string,
+	resourceVersion string,
+	asyncOperationId string,
+) (*operation.AsyncOperation, error) {
+	resp, err := c.client.DeleteNexusEndpoint(c.ctx, &cloudservice.DeleteNexusEndpointRequest{
+		EndpointId:       endpointID,
+		ResourceVersion:  resourceVersion,
+		AsyncOperationId: asyncOperationId,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return resp.AsyncOperation, nil
+}
+
+func NewNexusCommand(getNexusClientFn GetNexusClientFn) (CommandOut, error) {
+	var c *NexusClient
+	endpointNameFlag := &cli.StringFlag{
+		Name:     "name",
+		Aliases:  []string{"n"},
+		Usage:    "Endpoint name",
+		Required: true,
+	}
+	targetNamespaceFlag := &cli.StringFlag{
+		Name:     "target-namespace",
+		Aliases:  []string{"tns"},
+		Usage:    "Namespace in which a handler worker will be polling for Nexus tasks on",
+		Required: true,
+	}
+	targetNamespaceFlagOptional := &cli.StringFlag{
+		Name:     targetNamespaceFlag.Name,
+		Aliases:  targetNamespaceFlag.Aliases,
+		Usage:    targetNamespaceFlag.Usage + " (optional)",
+		Required: false,
+	}
+	targetTaskQueueFlag := &cli.StringFlag{
+		Name:     "target-task-queue",
+		Aliases:  []string{"ttq"},
+		Usage:    "Task Queue in which a handler worker will be polling for Nexus tasks on",
+		Required: true,
+	}
+	targetTaskQueueFlagOptional := &cli.StringFlag{
+		Name:     targetTaskQueueFlag.Name,
+		Aliases:  targetTaskQueueFlag.Aliases,
+		Usage:    targetTaskQueueFlag.Usage + " (optional)",
+		Required: false,
+	}
+	allowNamespaceFlag := &cli.StringSliceFlag{
+		Name:     "allow-namespace",
+		Aliases:  []string{"ans"},
+		Usage:    "Namespace that is allowed to call this endpoint",
+		Required: true,
+	}
+	namespaceFlag := &cli.StringSliceFlag{
+		Name:     "namespace",
+		Aliases:  []string{"ns"},
+		Usage:    "Namespace that is allowed to call this endpoint",
+		Required: true,
+	}
+	return CommandOut{
+		Command: &cli.Command{
+			Name:    "nexus",
+			Aliases: []string{"nxs"},
+			Usage:   "Commands for managing Nexus operations (EXPERIMENTAL)",
+			Before: func(ctx *cli.Context) error {
+				var err error
+				c, err = getNexusClientFn(ctx)
+				return err
+			},
+			Subcommands: []*cli.Command{
+				{
+					Name:        "endpoint",
+					Aliases:     []string{"ep"},
+					Usage:       "Commands for managing Nexus Endpoints (EXPERIMENTAL)",
+					Description: "Endpoint commands enable managing Nexus Endpoints",
+					Subcommands: []*cli.Command{
+						{
+							Name:        "get",
+							Aliases:     []string{"g"},
+							Usage:       "Get a Nexus Endpoint by name (EXPERIMENTAL)",
+							Description: "This command gets a Nexus Endpoint configuration by name from the Server",
```

Comment:
I think we should probably differ in the suffix here 'from the Cloud Account' might be better than `from the Server`

Reply:
Replaced here and in the other places ✔️